app/views/comments/_edit.html.erb

```diff
@@ -12,119 +12,48 @@
     <!-- toolbar needs location & comment_id to make unique element IDs -->
     <%= render :partial => "editor/toolbar", :locals => { :comment_id => comment.id.to_s, :location => :edit } %>
     
-    <div id="c<%= comment.id%>div" class="form-group">
-    <textarea aria-label="Edit Comment" onFocus="editing=true" name="body" class="form-control" id="c<%= comment.id%>text" rows="6" cols="40" placeholder="<%= placeholder %>"><%= !(comment.is_a?Answer) ? comment.comment : comment.content %></textarea>
-    <div class="imagebar">
-      <div id="c<%= comment.id%>progress" style="display:none;" class="progress progress-bar-container active pull-right">
-        <div id="c<%= comment.id%>progress-bar" class="progress-bar progress-bar-striped progress-bar-animated" style="width: 0%;"></div>
-      </div>
-      <p>
-        <span id="c<%= comment.id%>uploading" class="uploading uploading-text" style="display:none;">
-          <%= translation('comments._edit.uploading') %>
-        </span>
-        <span id="c<%= comment.id%>prompt" class="prompt choose-one-prompt-text">
-          <span style="padding-right:4px;float:left;" class="hidden-xs">
-          <%= raw translation('comments._edit.drag_and_drop') %>
+    <div id="c<%= comment.id%>div" class="form-group dropzone">
```

Comment:
Add the class `.dropzone` here because this is what `editor.js` uses to toggle preview.

Reply:
This seems a little odd sounding -  isn't `dropzone` supposed to mean the area we can drag a file/image into? Coudl this be an artifact from when preview and drag-drop were implemented at the same time, and might we think of a more legible classname or of separating preview and drag-drop? 

It kind of makes sense that we remove `.dragdrop` when we turn on preview, maybe... so you can't dragdrop onto the preview... but I don't know, shouldn't it just hide the dragdrop and not remove the class? This suggests to me that we should separate the functions and namings to make it more legible and less interdependent, whether or not I'm really understanding what's happening in the current model. What do you think?

Thanks, @noi5e !!!

Reply:
@jywarren I am totally in agreement that `.dropzone` is not really the best class to refer to in this instance. I think the reason this was done in the past has to do with this:
![103472308-f5ae0400-4d40-11eb-8d18-88099b4addfd](https://user-images.githubusercontent.com/4361605/105936040-dc2e6e00-6007-11eb-8b42-6c9eccd138ea.jpg)

What we're toggling on is `B` in the above image. I think in the past, `.dropzone` was toggled for preview because it was just a way of referring to _"The visible, largest part of the comment form"_. I acknowledge it's weird, but I was just following the precedent of older code.

For example, it's happening here in this legacy code in `editor.js`:
https://github.com/publiclab/plots2/blob/b3009791076b15c9da81d58aec3de2a7a2b0f784/app/assets/javascripts/editor.js#L141-L146

I think the best thing to do moving forward is to add another CSS ID/class 😢 that has the same coverage as `.dropzone` in this instance, and toggle preview on **that**. Adding it to my planning issue so I don't forget to do this.

Reply:
Oh, thanks for the explanation! This makes total sense and is the kind of thing that is hard to plan for in the moment, but is really helpful down the road later. A second class sounds like just the right solution!